files/ko/web/javascript/reference/operators/conditional_operator/index.html

```diff
@@ -22,169 +29,68 @@ <h2 id="구문">구문</h2>
-</pre>
-
-<p>하나의 케이스 당 둘 이상의 단일 작업을 수행하려면 쉼표로 구분하고 괄호로 묶으세요.</p>
+<p>이와같이 <code>null</code> 값을 처리할 때에도 일반적으로 사용됩니다.:</p>
 
-<pre class="brush: js notranslate">var stop = false, age = 23;
+<pre class="brush: js">let greeting = person =&gt; {
+  let name = person ? person.name : `stranger`
+  return `Howdy, ${name}`
+}
 
-age &gt; 18 ? (
-    alert("OK, you can go."),
-    location.assign("continue.html")
-) : (
-    stop = true,
-    alert("Sorry, you are much too young!")
-);
+console.log(greeting({name: `Alice`}));  // "Howdy, Alice"
+console.log(greeting(null));             // "Howdy, stranger"
 </pre>
 
-<p>또한, 값을 할당하는 동안 하나 이상의 연산도 가능합니다. 이 경우에, 괄호 안의 값중 마지막 쉼표 (,) 다음의 값이 최종 할당 값이 됩니다.</p>
-
-<pre class="brush: js notranslate">var age = 16;
-
-var url = age &gt; 18 ? (
-    alert("OK, you can go."),
-    // alert returns "undefined", but it will be ignored because
-    // isn't the last comma-separated value of the parenthesis
-    "continue.html" // the value to be assigned if age &gt; 18
-) : (
-    alert("You are much too young!"),
-    alert("Sorry :-("),
-    // etc. etc.
-    "stop.html" // the value to be assigned if !(age &gt; 18)
-);
-
-location.assign(url); // "stop.html"</pre>
-
 
+<h3 id="Conditional_chains">연속된 조건문 처리하기</h3>
 
-<h3 id="삼항_연산자로_반환하기">삼항 연산자로 반환하기</h3>
+<p>삼항 연산자는 <code>if … else if … else if … else</code>와 같은 "연속된 조건"을 사용할 수 있습니다.</p>
 
-<p>삼항 연산자는 <code>if / else</code> 문을 사용하는 함수를 반환하는 데 적합합니다.</p>
-
-<pre class="notranslate"><code>var func1 = function( .. ) {
-  if (condition1) { return value1 }
-  else { return value2 }
+<pre class="brush: js">function example(…) {
+    return condition1 ? value1
+         : condition2 ? value2
+         : condition3 ? value3
+         : value4;
 }
 
-var func2 = function( .. ) {
-  return condition1 ? value1 : value2
-}</code></pre>
-
-<p>다음과 같이 법적으로 술을 마실수 있는지 여부를 반환하는 함수를 만들수 있습니다.</p>
+// Equivalent to:
 
-<pre class="notranslate"><code>function canDrinkAlcohol(age) {
-  return (age &gt; 21) ? "True, over 21" : "False, under 21";
+function example(…) {
+    if (condition1) { return value1; }
+    else if (condition2) { return value2; }
+    else if (condition3) { return value3; }
+    else { return value4; }
 }
-var output = canDrinkAlcohol(26);
-console.log(output); // "True, over 21"</code></pre>
-
-<p><code>if / else</code> 문을 대체하는 삼항연산자가 <code>return</code>을 여러 번 사용하고 if 블록 내부에서 한줄만 나올때 <code>return</code>을 대체 할 수 있는 좋은 방법이됩니다.</p>
-
-<p>삼항연산자를 여러 행으로 나누고 그 앞에 공백을 사용하면 긴 <code>if / else</code> 문을 매우 깔끔하게 만들 수 있습니다. 이것은 동일한 로직을 표현하지만 코드를 읽기 쉽게 만들어 줍니다.</p>
-
-<pre class="notranslate"><code>var func1 = function( .. ) {
-  if (condition1) { return value1 }
-  else if (condition2) { return value2 }
-  else if (condition3) { return value3 }
-  else { return value4 }
-}
-
-var func2 = function( .. ) {
-  return condition1 ? value1
-       : condition2 ? value2
-       : condition3 ? value3
-       :              value4
-}</code>
 </pre>
 
 <h2 id="명세">명세</h2>
```

Comment:
아이디는 영어로 표기 부탁드리겠습니다.
```suggestion
<h2 id="Specifications">명세</h2>
```